### PR TITLE
Yifan-project-weather-app

### PR DESCRIPTION
**Pull request by Yifan-858**
## Netlify link
https://yifan-weather-app.netlify.app/

## Collaborators
[Yifan-858]

### CODE REVIEW

**Review by El1an3 — COMMENTED** [review]
Such a good job, Yifan! You should be very proud!
From what I've seen, all the requirements have been met, including the stretch goals. Your code is very clean and clear. Your documentation guides through the code beautifully. I really like it and wish mine was as organised. 

If I had to be picky, I'd say the contrast for the search bar and my location button is a bit low, but that's just me looking for flaws. I would probably have set a maximum width for the desktop so that the app doesn't take up the whole page.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+  --colorTheme-drizzle-background: #b8af83;
+  --colorTheme-drizzle-text: #004f22;
+  --colorTheme-thunderstorm-background: #ee573f;
+  --colorTheme-thunderstorm-text: #000000;
+  --colorTheme-other-background: #786060;
+  --colorTheme-other-text: #fffc9c;
+
+  --search--background: #dbdbdb36;
+  --search--prompt: #ffffff9a;
+  --search--hover: #ffffffe6;
```
Comment (HIPPIEKICK) [inline style.css:21]: This doesn't show so well, maybe decrease opacity? 🔍

File: style.css [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+}
+
+body {
+  font-family: "Montserrat", sans-serif;
+  background-color: #0b3338;
+  color: #fff;
+  transition: background-color 0.2s color 0.2s;
+}
+
+.clearTheme {
```
Comment (HIPPIEKICK) [inline style.css:37]: Class names should be named in kebab-case

**Review by HIPPIEKICK — APPROVED** [review]
Good job with your weather app! Wrote some small things to think about going forward. Keep it up 🌞

**Pull request closed without merging**